diff --git a/node_modules/origami-build-tools/lib/helpers/files.js b/node_modules/origami-build-tools/lib/helpers/files.js
index 78a4291..533096e 100644
--- a/node_modules/origami-build-tools/lib/helpers/files.js
+++ b/node_modules/origami-build-tools/lib/helpers/files.js
@@ -170,6 +170,7 @@ function getSassTestFiles(cwd) {
  */
 function getSassIncludePaths (cwd, config = {sassIncludePaths: []}) {
 	const npmPaths = [
+		path.join('..', '..', 'node_modules'),
 		'node_modules',
 		path.join('node_modules', '@financial-times')
 	];
diff --git a/node_modules/origami-build-tools/lib/tasks/demo.js b/node_modules/origami-build-tools/lib/tasks/demo.js
index 4e64c2d..88979bb 100644
--- a/node_modules/origami-build-tools/lib/tasks/demo.js
+++ b/node_modules/origami-build-tools/lib/tasks/demo.js
@@ -59,7 +59,7 @@ module.exports = function (cfg) {
 		renderer: ListrRenderer,
 		collapse: false,
 		showSubtasks: true,
-		concurrent: true
+		concurrent: false
 	})
 		.run();
 };
diff --git a/node_modules/origami-build-tools/lib/tasks/install-npm.js b/node_modules/origami-build-tools/lib/tasks/install-npm.js
index 56a9e62..5913a87 100644
--- a/node_modules/origami-build-tools/lib/tasks/install-npm.js
+++ b/node_modules/origami-build-tools/lib/tasks/install-npm.js
@@ -29,7 +29,7 @@ module.exports = function (cfg) {
 				renderer: ListrRenderer,
 				collapse: false,
 				showSubtasks: true,
-				concurrent: true
+				concurrent: false
 			});
 		},
 		skip: async function () {
diff --git a/node_modules/origami-build-tools/lib/tasks/install.js b/node_modules/origami-build-tools/lib/tasks/install.js
index 275a3ff..713a2f9 100644
--- a/node_modules/origami-build-tools/lib/tasks/install.js
+++ b/node_modules/origami-build-tools/lib/tasks/install.js
@@ -16,6 +16,6 @@ module.exports = function (cfg) {
 		renderer: ListrRenderer,
 		collapse: false,
 		showSubtasks: true,
-		concurrent: true
+		concurrent: false
 	}).run();
 };
diff --git a/node_modules/origami-build-tools/lib/tasks/karma.js b/node_modules/origami-build-tools/lib/tasks/karma.js
index aad3ffa..dce00af 100644
--- a/node_modules/origami-build-tools/lib/tasks/karma.js
+++ b/node_modules/origami-build-tools/lib/tasks/karma.js
@@ -116,7 +116,7 @@ module.exports = (cfg) => {
 				renderer: ListrRenderer,
 				collapse: false,
 				showSubtasks: true,
-				concurrent: true
+				concurrent: false
 			});
 		},
 		skip: () => {
diff --git a/node_modules/origami-build-tools/lib/tasks/test-sass-compilation.js b/node_modules/origami-build-tools/lib/tasks/test-sass-compilation.js
index 319c35a..e7d0152 100644
--- a/node_modules/origami-build-tools/lib/tasks/test-sass-compilation.js
+++ b/node_modules/origami-build-tools/lib/tasks/test-sass-compilation.js
@@ -116,7 +116,7 @@ module.exports = function (cfg) {
 				renderer: ListrRenderer,
 				collapse: false,
 				showSubtasks: true,
-				concurrent: true
+				concurrent: false
 			});
 		},
 		skip: async () => {
diff --git a/node_modules/origami-build-tools/lib/tasks/test.js b/node_modules/origami-build-tools/lib/tasks/test.js
index 7f6ae58..84d462d 100644
--- a/node_modules/origami-build-tools/lib/tasks/test.js
+++ b/node_modules/origami-build-tools/lib/tasks/test.js
@@ -24,7 +24,7 @@ module.exports = function (cfg) {
 			renderer: ListrRenderer,
 			collapse: false,
 			showSubtasks: true,
-			concurrent: true,
+			concurrent: false,
 			exitOnError: false
 		}
 	).run();
diff --git a/node_modules/origami-build-tools/lib/tasks/verify.js b/node_modules/origami-build-tools/lib/tasks/verify.js
index a69b55a..b738958 100644
--- a/node_modules/origami-build-tools/lib/tasks/verify.js
+++ b/node_modules/origami-build-tools/lib/tasks/verify.js
@@ -27,7 +27,7 @@ module.exports = function(cfg) {
 			renderer: ListrRenderer,
 			collapse: false,
 			showSubtasks: true,
-			concurrent: true,
+			concurrent: false,
 		}
 	).run();
 };
